scripts/close_inactive.rb

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env ruby
+require 'octokit'
+
+REPOS = ["theforeman/foreman"]
+KEEP = 6 # months
+THRESHOLD = DateTime.now << KEEP
+COMMENT = <<EOC
+This PR has been inactive for #{KEEP} months, closing for now.
+Feel free to reopen when you return to it.
+Thank you for your contribution, @%s!
+EOC
+LABELS = ["Inactive"]
+
+c = Octokit::Client.new(:access_token => ENV['GITHUB_OAUTH_TOKEN'])
+REPOS.each do |repo|
+  c.pull_requests(repo).each do |pr|
+    number = pr[:number]
+    user = pr[:user][:login]
+    updated_at = pr[:updated_at].to_datetime
+    if updated_at < THRESHOLD
+      puts "Closing PR #{number} with latest update #{updated_at}"
+      #c.add_comment(repo, number, COMMENT % user)
+      #c.add_labels_to_an_issue(repo, number, LABELS)
```

Comment:
> These should be enabled or be configurable, not commented.

Sure, my idea is to push it commented and inspect the log before
commenting them out.

-- 
Later,
 Lukas #lzap Zapletal